liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -994,6 +994,12 @@ public void installFeatures(File configFile, File serverDir) {
                 }
             } catch (MojoExecutionException e) {
                 log.error("Failed to install features from configuration file", e);
+                if (generateFeatures && !getEsaDependency(project.getDependencies()).isEmpty()) {
```

Comment:
Should we explicitly check for the "feature conflict" error from install features to display this warning, not just any install feature failure?

Reply:
Good idea, done